packages/sg-basic-whitelist/src/lib.rs

```diff
@@ -0,0 +1,2 @@
+// re-export the proc macros
+pub use sg_basic_whitelist_derive::sg_basic_whitelist_query;
```

Comment:
i havn't seen this before, where there a nested Cargo.toml packages within a workspace, maybe we put the macro Cargo.toml up one level?